### PR TITLE
[OPPRO-319] Add one_step_veloxbackend.sh document

### PR DESCRIPTION
**Pull request by zhejiangxiaomai**
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)

(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

### CODE REVIEW

File: docs/GlutenUsage.md [hunk 1]
```diff
@@ -1,20 +1,27 @@
...
+| build_arrow_from_source   | pull the source code and build arrow lib| ON|
+| build_velox_from_source   | pull the source code and build velox lib| ON|
 
 There are some parameters can be set via -- with build_arrow_for_gazelle.sh.
 
 | Parameters | Description | Default value |
 | ---------- | ----------- | ------------- |
 | arrow_home | Arrow build path            | GLUTEN_DIR/ep/build-arrow/build|
-| build_type | ARROW build type            | release|
-| enable_ep_cache |  Use cache lib when arrow is not change |  OFF   |
```
Comment (zhztheplayer) [inline docs/GlutenUsage.md:17]: Is there a way to enable ep cache from one step script?

**Comment by zhztheplayer** [comment]
Is it possible to call one step script in GHA ci? So the script can be covered by our daily testing.

Although it's off-topic so we can raise another PR for that if OK.

File: docs/GlutenUsage.md (hunk 1 above)
Comment (zhejiangxiaomai) [reply docs/GlutenUsage.md:17]: Sorry, I forgot. will add enable_ep_cache in one_step_veloxbackend.sh.

**Comment by zhejiangxiaomai** [comment]
> Is it possible to call one step script in GHA ci? So the script can be covered by our daily testing.
> 
> Although it's off-topic so we can raise another PR for that if OK.

OK,  I can do this, In fact, jenkens nightly test use one_step_veloxbackend.sh now.

**Review by FelixYBW — APPROVED** [review]

**Pull request merged**